### PR TITLE
Minor fixes to the user API

### PR DESCRIPTION
**Pull request by prashbnair**
This PR fixes
- Invalid Synk Token HTML in response error
- Invalid Default value for `ENCRYPTION_KEY_FOR_SNYK_TOKEN` in config files
- Empty Payload: Server error in user [PUT] API.

### CODE REVIEW

File: bayesian/default_config.py [hunk 1]
```diff
@@ -48,7 +48,8 @@
 
 SNYK_API_TOKEN_VALIDATION_URL = os.getenv('SNYK_API_TOKEN_VALIDATION_URL',
                                           'https://snyk.io/api/v1/verify/token')
-ENCRYPTION_KEY_FOR_SNYK_TOKEN = os.getenv('ENCRYPTION_KEY_FOR_SNYK_TOKEN', 'SNYK')
+ENCRYPTION_KEY_FOR_SNYK_TOKEN = os.getenv('ENCRYPTION_KEY_FOR_SNYK_TOKEN',
```
Comment (deepak1725) [inline bayesian/default_config.py:51]: `user[PUT]` API is not using this value here but one in https://github.com/fabric8-analytics/fabric8-analytics-utils/blob/e2207358ecb8289e2e941b0011d8b86c848e8a21/f8a_utils/default_config.py#L6

Comment (deepak1725) [reply bayesian/default_config.py:51]: Although it's good to have this change here too.

File: bayesian/api/user_api.py [hunk 2]
```diff
@@ -40,17 +42,20 @@ def generate_uuid_for_user():
 def create_or_update_user():
     """Endpoint for creating or updating user details."""
     content = request.json
```
Comment (deepak1725) [inline bayesian/api/user_api.py:44]: can we check for empty payload here..?

**Review by deepak1725 — CHANGES_REQUESTED** [review]

File: bayesian/default_config.py (hunk 1 above)
Comment (prashbnair) [reply bayesian/default_config.py]: Done

File: bayesian/api/user_api.py (hunk 2 above)
Comment (prashbnair) [reply bayesian/api/user_api.py:44]: It is being checked.  The content is null condition takes care of it.

Comment (deepak1725) [reply bayesian/api/user_api.py:44]: Sorry, I missed it.

**Review by deepak1725 — APPROVED** [review]
lgtm

**Pull request merged**